### ISSUE TITLE
[DOC] Broken hover effect in the deprecated light mode.

### ISSUE THREAD

**Issue by sarojregmi200** [refinedev/refine#4666]
### Documentation issue

While hovering the deprecated items in the table, the hover effect makes the tool tip unreadable. Because it is transparent. 

Links: [Documentation refine config](https://refine.dev/docs/api-reference/core/components/refine-config/#api-reference)

![image](https://github.com/refinedev/refine/assets/94213188/09dbfe05-79b3-45be-a456-add514eadd0c)

[Documentation `useCheckboxGroup` page](https://refine.dev/docs/api-reference/antd/hooks/field/useCheckboxGroup/#sort).

![image](https://github.com/refinedev/refine/assets/94213188/4ab4d93a-73db-4c94-9f97-4b620bdd0d8c)

I have verified the issue by opening the documentation in incognito mode. And it exists there also.

### Describe the thing to improve

The css variable that is used to switch the background of the tool tip during the theme change, changes the variable value to `#1b1b1d` in dark mode and changes to `#0000` in light mode, which means instead of giving a white color it is giving a completely black color with 0 as it's alpha value. So it is a transparent color Hence the problem is caused. 

### Describe the solution (optional)

While changing to light mode it should contain a white color.

**Comment by sarojregmi200** [comment]
Apparently I am not able to find the exact `css` file that contains the code, but when I compile it give me   `--ifm-background-color: #0000;` which is the error.

**Comment by sarojregmi200** [comment]
After searching and digging for a while I am still not able to find the `css` file in the project, that has the same code that we receive in the browser. I want to solve this issue, So, can anyone help me out with this and confirm that they are also facing similar issue with the documentation.

**Comment by salihozdemir** [comment]
Hey @sarojregmi200,

Thank you for your report.

You can change the tooltip css on this file:

https://github.com/refinedev/refine/blob/next/documentation/src/components/tooltip/styles.module.css

You can change like this the file:

```css
.tooltip {
    z-index: var(--ifm-z-index-overlay);
    @apply relative flex flex-col items-center;
}

.tooltipContainer {
    @apply absolute bottom-0 left-0 flex flex-col items-center invisible mb-6 transition-all ease-out delay-75 border rounded-sm shadow-lg bg-gray-0 dark:bg-gray-900 border-gray-200 dark:border-gray-700;
}

.tooltipContent {
    @apply relative p-2 w-60;
}

.tooltipArrow {
    @apply absolute bottom-0 w-3 h-3 rotate-45 translate-y-2/4 bg-gray-0 dark:bg-gray-900 border-r border-b border-gray-200 dark:border-gray-700;
}

```

**Comment by sarojregmi200** [comment]
Hey @salihozdemir.
Thank you for pointing me to the correct file, I inspected the problem more deeply and found out that the background variable `--ifm-background-color` was out of reach when it was modified by the theme change. 

```
[data-theme="light"] {
    --ifm-background-color: #fff;
}
```

so, I added the same theme variable change method in the file you mentioned rather than rewritting the styles. It is a good solution I guess.
